config/kafka-monitor.properties

```diff
@@ -77,23 +77,23 @@
   },
 
 
-  "reporter-service": {
-    "class.name": "com.linkedin.kmf.services.DefaultMetricsReporterService",
-    "report.interval.sec": 1,
-    "report.metrics.list": [
-      "kmf:type=kafka-monitor:offline-runnable-count",
-      "kmf.services:type=produce-service,name=*:produce-availability-avg",
-      "kmf.services:type=consume-service,name=*:consume-availability-avg",
-      "kmf.services:type=produce-service,name=*:records-produced-total",
-      "kmf.services:type=consume-service,name=*:records-consumed-total",
-      "kmf.services:type=consume-service,name=*:records-lost-total",
-      "kmf.services:type=consume-service,name=*:records-duplicated-total",
-      "kmf.services:type=consume-service,name=*:records-delay-ms-avg",
-      "kmf.services:type=produce-service,name=*:records-produced-rate",
-      "kmf.services:type=produce-service,name=*:produce-error-rate",
-      "kmf.services:type=consume-service,name=*:consume-error-rate"
-    ]
-  }
+#  "reporter-service": {
```

Comment:
Can we include the existing "reporter-service" in the default kafka-monitor.properties? I think most users probably would want to see the metrics in the console by default.